Global/VisualStudioCode.gitignore

```diff
@@ -3,3 +3,4 @@
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.history
```

Comment:
Shouldn't this be inside `.vscode/`? If so, it should be covered by the rule on line 1

Reply:
no the folder is outside of .vscode

Reply:
I'd like to see some documentation that explains this file better - describing the feature is one thing, but it'd be nice to confirm we're looking at the right file in the right location before merging this in.

It also seems suboptimal for this to not be using the `.vscode` folder, especially if it's an in-the-box feature of VSCode, for what looks like a generated file.